network/f5/bigip_virtual_server.py

```diff
@@ -508,6 +509,15 @@ def set_default_persistence_profiles(api, name, persistence_profile):
                 profiles=[[{'profile_name': persistence_profile, 'default_profile': True}]]
             )
             updated = True
+        # If a virtual server has a persistence profile and would like this to be None this section enabled this
+        # When persistence_profile is not defined in task or set to Null in vars the persistence profile will be
+        # set to None.
+        if persistence_profile is None or False:
```

Comment:
@mhite what are your thoughts on a separate param to zero out those persistence profiles in a similar fashion to the `user` module's use of a binary parameter for generating ssh keys

https://github.com/ansible/ansible-modules-core/blob/devel/system/user.py#L143